specifications/network/onchain-discovery.md

```diff
@@ -58,7 +58,7 @@ enum ConsensusScheme {
 
 * [`AccountAddress`](../common/data_structures.md#accountaddress)
 * [`RawNetworkAddress`](network-address.md)
-* [`RawEncNetworkAddress`](network-address.md)
+* [`RawNetworkAddress`](network-address.md)
```

Comment:
I don't think `RawNetworkAddress` still exists.  One of those old unfixed docs problems.  In a separate commit (but same PR), flip them to `NetworkAddress`.  This includes in the `ValidatorConfig` comment above

Reply:
Can you remove the duplicate line?

Since you renamed it, now it's the same as the line above it.